share/github-backup-utils/version

```diff
@@ -1 +1 @@
-2.22.0
+3.0.0.rc1
```

Comment:
im not sure if this should be `3.0.0.rc1` or just `3.0.0`. I believe this is just used to compare against the version of the instance that's being backed up so I don't think it has any concept of `gm`s or `rc`s.